# try that many times, in a row, to find a goal
max_goal_searches: 5

# robot has that many seconds to reach a goal
goal_timeout: 20.0

# how far from a goal the robot hat to be
# to considered visited (in meters) 
goal_reached_dist: 0.8
